src/i18n/index.ts

```diff
@@ -2,6 +2,7 @@ import enUS from './en-US';
 import zh from './zh';
 import zhTW from './zh-TW';
 import ja from './ja';
+import fr from './fr';
```

Comment:
You can still use `index.ts` for `fr/fr.ts`. The import name should be `fr` (like now)